### DISCUSSION TITLE
SCIRun5 command line interface

### DISCUSSION THREAD

**Discussion by salmanfarsi100** [SCIInstitute/SCIRun#2424] [Q&A] [answered]
Hi! I am trying to save all ViewScene images in a pipeline. The pipeline runs smoothly without any problems, but I don't see any resulting images in the folder where my SCIRun file is. This is the command that I enter in the terminal. 
`"C:\Program Files\SCIRun_5.0\bin\SCIRun.exe" -E visualize_E_field.srn5 --save-images `
Am I missing something?

**Comment by jessdtate** [comment, marked as answer]
The images are saved in a default spot, usually `~/scirun5screenshots`.  Did you check there?  you can also change the directory the screenshots save to in the preferences, or in the viewscene (the `ScreenshotDirectory` statekey)

**Reply by salmanfarsi100** [reply]
So, I set the screenshot path in the ViewScene module. The image is getting saved now, but it's a black screen instead of the actual view. How do I get the actual view in the saved image?
